service/src/main/kotlin/fi/espoo/evaka/varda/VardaFeeData.kt

```diff
@@ -244,8 +244,8 @@ private fun baseToFeeData(h: Handle, feeDataBase: VardaFeeDataBase, personServic
         .map { guardian ->
             VardaGuardian(
                 ssn = guardian.identity.toString(),
-                firstName = guardian.firstName!!,
-                lastName = guardian.lastName!!
+                firstName = guardian.firstName ?: "etunimi_puuttuu",
+                lastName = guardian.lastName ?: "sukunimi_puuttuu"
```

Comment:
Not sure how Varda will behave if we send incorrect names. I wouldn't try to send incorrect names. Maybe just filter out and log the cases where guardian names are empty